.github/workflows/build-deploy-pudl.yml

```diff
@@ -9,7 +9,6 @@ on:
 
 env:
   GCP_BILLING_PROJECT: ${{ secrets.GCP_BILLING_PROJECT }}
-  BUILD_REF: ${{ github.ref_name }} # This is changed to dev if running on a schedule
```

Comment:
I think we can pretty much get rid of the `BUILD_REF` variable now, since we don't have to do the "check out one branch when the workflow is running on another" thing.

Reply:
Yes! I think so too.